src/components/Footer/Footer.css.ts

```diff
@@ -0,0 +1,39 @@
+import { style } from '@vanilla-extract/css';
+
+export const container = style({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-around',
+  height: '130px',
+  padding: '0 30px',
+  backgroundColor: 'rgb(0, 0, 0, 0.9)',
+  color: 'white',
+});
+
+export const topFooter = style({
+  display: 'flex',
+  justifyContent: 'space-around',
+});
+
+export const categoryList = style({
+  display: 'flex',
+  listStyle: 'none',
+  fontSize: '15px',
+  fontWeight: 'bold',
+  alignItems: 'center',
+});
+
+export const category = style({
+  width: '100px',
+});
```

Comment:
```suggestion
export const categoryList = style({
  display: 'flex',
  listStyle: 'none',
  fontSize: '15px',
  fontWeight: 'bold',
  alignItems: 'center',
  gap: '50px',
});

export const category = style({
  width: 'fit-content',
});
```

이렇게 하는 것은 어떤가요?
100px로 width를 잡아버리니 글자가 짧은 요소는 간격이 넓게 보이더라구요ㅠㅠ

Reply:
오 좋습니다! 수정했습니다. [a9ab415](https://github.com/Kernel360/F2-omocha-FE/pull/5/commits/a9ab415b39751ef4227f90ce1b263689ef7ee0fb)